doc/ref/helpintf.xml

```diff
@@ -90,25 +90,20 @@ any more. The book can be re-installed using <Ref Func="HELP_ADD_BOOK" />. <P/>
 <Section Label="The manual.six File">
 <Heading>The manual.six File</Heading>
 
-If a <F>manual.six</F> file for a help book is not in the format of the
-<F>gapmacro.tex</F> macros, explained in the document 
-<Q>The gapmacro.tex Manual Format</Q> (see the file 
-<F>gap4r5/doc/gapmacrodoc.pdf</F> from the tools archive 
```

Comment:
Test, I want to throw that away (and also the reference to the dir name `gap4r5`). After all, this file is only of interest for authors of legacy packages, who already should know about it, or be able to find it, or be able to ask us where it is. But I do not want new GAP developers to accidentally stumbler over it and perhaps even get the wrong idea that using it is encouraged.